src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,23 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        <UserInfo user={user} />
```

Comment:
user can be null, let's handle this case
```suggestion
        {user && <UserInfo user={user} />}
```